source/Background/services/DashService.ts

```diff
@@ -112,6 +129,24 @@ class DashService {
     return account.getTransactionHistory();
   }
 
+  async getUnusedAddress(): Promise<string> {
+    const account = await this._wallet.getAccount({synchronize: false});
+    return account.getUnusedAddress();
+  }
+
+  async createTransaction({
+    recipient,
+    satoshis,
+  }: CreateTransactionPayload): Promise<void> {
+    const account = await this._wallet.getAccount({synchronize: false});
+    return account.createTransaction({recipient, satoshis});
+  }
+
+  async broadcastTransaction(transaction: TransactionInfo): Promise<string> {
```

Comment:
Instead of `TransactionInfo`, make it a `string`, and at the UI side do `transaction.toString('hex')` (Or `new Transaction(transaction).toString('hex')` in case `transaction` is not an instance of `Transaction`.